gradle-suppressible-error-prone/src/test/groovy/com/palantir/gradle/suppressibleerrorprone/CodeChangesTest.groovy

```diff
@@ -0,0 +1,217 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.suppressibleerrorprone
+
+import com.google.common.base.Splitter
+import one.util.streamex.StreamEx
+import spock.lang.Unroll
+
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.stream.Collectors
+import java.util.stream.Stream
+
+class CodeChangesTest extends AbstractSuppressibleErrorPronePluginIntegrationTest {
```

Comment:
Best to look at [`AddNewSuppressionsAtEveryLevel.java`](https://github.com/palantir/suppressible-error-prone/pull/39/files#diff-30997f42a0f6268796064a71edc68fd4810b56ce453258fe040218d8da3d163f) and on to get an idea of what the test for this look like